src/command-line/index.js

```diff
@@ -58,12 +68,12 @@ require("./install");
 
 // TODO: Remove this when releasing The Lounge v3
 if (process.argv[1].endsWith(`${require("path").sep}lounge`)) {
-	log.warn(`The ${colors.red("lounge")} CLI is ${colors.bold("deprecated")} and will be removed in v3.`);
+	log.warn(`The ${colors.red("lounge")} CLI is ${colors.bold.red("deprecated")} and will be removed in v3.`);
 	log.warn(`Use ${colors.green("thelounge")} instead.`);
 	process.argv[1] = "thelounge";
 }
 
-program.parse(process.argv);
+program.parse(argvWithoutOptions.args.concat(argvWithoutOptions.unknown));
```

Comment:
Add a comment here, it's not very clear where `unknown` comes from.

Reply:
Done, thanks for pointing this out!